models/models.go

```diff
@@ -10,58 +10,58 @@ import (
 
 // Filter represents a structure for a filter job
 type Filter struct {
-	InstanceID       string      `json:"instance_id"`
-	DimensionListURL string      `json:"dimension_list_url,omitempty"`
-	Dimensions       []Dimension `json:"dimensions,omitempty"`
-	Downloads        Downloads   `json:"downloads,omitempty"`
-	Events           Events      `json:"events,omitempty"`
-	FilterID         string      `json:"filter_job_id,omitempty"`
-	State            string      `json:"state,omitempty"`
-	Links            LinkMap     `json:"links,omitempty"`
+	InstanceID       string      `bson:"instance_id"          json:"instance_id"`
+	DimensionListURL string      `bson:"dimension_list_url"   json:"dimension_list_url,omitempty"`
+	Dimensions       []Dimension `bson:"dimensions,omitempty" json:"dimensions,omitempty"`
+	Downloads        Downloads   `bson:"downloads"            json:"downloads,omitempty"`
+	Events           Events      `bson:"events"               json:"events,omitempty"`
```

Comment:
Again probably want events bson tag to be omitempty too

Reply:
done